### ISSUE TITLE
Compatibility with swift 4.1 and XCode 9.4.1

### ISSUE THREAD

**Issue by mirnpnh** [EurekaCommunity/CreditCardRow#9]
Hi,
Will it be possible to see this updated to add compatibility to those new versions? I was trying to update my project with this new version and CreditCardRow was depending on Swift 4.0 version of Eureka which made it impossible to do so.

Thanks in advance,

**Comment by mats-claassen** [comment]
The dependency is on Eureka ~> 4.0 so you should be able to install Eureka 4.2 with this if you run `pod update`.

CreditCardRow works for me in Swift 4.1 and Xcode 9.4

**Comment by mirnpnh** [comment]
Thanks for the response however when i try pod install without versions for Eureka and creditCardRow it automatically retrieves v4 of Eureka which is incompatible with swift 4.1. When i set their version on Eureka4.2.0 and CreditCardRow3.0.0 it throws me this error which shows CreditCardRow can only work with Eureka4.0.0.
```
CocoaPods could not find compatible versions for pod "Eureka":
  In Podfile:
    CreditCardRow (~> 3.0.0) was resolved to 3.0.0, which depends on
      Eureka (= 4.0)

    Eureka (~> 4.2.0)
```

**Comment by mats-claassen** [comment]
Sorry, you are right. This was fixed after the last release. I will release a new version with this fix.

For now you can use the master branch

**Comment by mats-claassen** [comment]
Released version 3.1.0

**Comment by mirnpnh** [comment]
Thank you so much for your attention, I appreciate it 👍
